packages/function-tree/src/devtools/index.js

```diff
@@ -1,33 +1,28 @@
-import {FunctionTree} from '../'
 import WebSocket from 'universal-websocket-client'
 import Path from '../Path'
 const VERSION = 'v1'
 
-class DevtoolsClass {
-  constructor (tree, options = {
-    remoteDebugger: null
+export class Devtools {
```

Comment:
Due to Node

```js
import Devtools from 'function-tree/devtools'
const Devtools = require('function-tree/devtools').Devtools
```

Or you have to do: 

```js
const Devtools = require('function-tree/devtools').default
```

Which is super weird... so it is just a convention